nl/mail/chrome/messenger/safeMode.dtd

```diff
@@ -36,33 +36,32 @@
 #endif
    - ***** END LICENSE BLOCK ***** -->
 
-<!ENTITY changeAndRestartButton2.label     "Save and Restart">
+<!ENTITY changeAndRestartButton2.label     "Opslaan en Herstarten">
 <!ENTITY changeAndRestartButton2.accesskey "S">
 
 
-<!ENTITY safeModeOptionsWindow.width       "width: 49em;">
+<!ENTITY safeModeOptionsWindow.width       "breedte: 49em;">
 <!ENTITY safeModeOptionsDialog.title       "&brandShortName; Trouble Shooting">
 
-<!ENTITY safeModeOptionsDescription2.label "If you are experiencing difficulty with Postbox, try restarting 
-with one or more of these trouble shooting options selected:">
+<!ENTITY safeModeOptionsDescription2.label "Als u problemen verwacht met Postbox, probeer dan om te herstarten met een of meerdere trouble shooting opties geselecteerd:">
 
 
-<!ENTITY disableAddons.label              "Permamently Disable all Add-ons">
+<!ENTITY disableAddons.label              "Permament alle Add-ons uitschakelen">
 <!ENTITY disableAddons.accesskey          "D">
 
-<!ENTITY resetToolbars.label              "Reset User Interface">
+<!ENTITY resetToolbars.label              "Reset Gebruiker Interface">
 <!ENTITY resetToolbars.accesskey          "R">
 
-<!ENTITY resetIndex.label                 "Reset Search Index">
+<!ENTITY resetIndex.label                 "Reset Zoek Index">
 <!ENTITY resetIndex.accesskey             "I">
 
-<!ENTITY safeMode.label                   "Safe Mode (temporarily disables all Add-ons and custom settings)">
+<!ENTITY safeMode.label                   "Veilige Modus (schakelt tijdelijk alle Add-ons en aangepaste instellingen uit.)">
 <!ENTITY safeMode.accesskey               "M">
 
-<!ENTITY profileManager.label                   "Open Profile Manager on Restart">
+<!ENTITY profileManager.label                   "Open Profiel Manager bij Herstarten">
```

Comment:
CORRECTION NEEDED, it should be: "Open Profielmanager bij herstarten" or "Open profielmanager bij herstarten".